### DISCUSSION TITLE
Inserting mastodon verification in theme

### DISCUSSION THREAD

**Discussion by mikesplaces** [adityatelange/hugo-PaperMod#896] [Q&A] [answered]
Hi. I would like to verify my new site on mastodon. They require something like this to be added:

`<a rel="me" `href="https://photog.social/@mpmilestogo">Mastodon</a>`

Is there a method to override the theme to accomplish this?

**Comment by ghost** [comment, marked as answer]
Yes, you can add a file "footer.html" to layouts/partials/. 
Add the content of the original footer file and add your <a rel="me"></a>.

If you need more help contact me on chat
